docs/src/app/(public)/(content)/react/handbook/animation/page.mdx

```diff
@@ -0,0 +1,68 @@
+# Animation
+
+<Subtitle>A guide to animating Base UI components.</Subtitle>
+<Meta name="description" content="A guide to animating Base UI components." />
+
+Base UI components support CSS Transition, CSS Animation, or JavaScript animation libraries.
+
+## CSS Transition
+
+Many Base UI components expose `[data-starting-style]` and `[data-ending-style]` attributes to use as style hooks for CSS Transition.
+
+```css title="popover.css"
+.Popup {
+  box-sizing: border-box;
+  padding: 1rem 1.5rem;
+  background-color: canvas;
+  transform-origin: var(--transform-origin);
+  transition:
+    transform 150ms,
+    opacity 150ms;
+
+  &[data-starting-style],
+  &[data-ending-style] {
+    opacity: 0;
+    transform: scale(0.9);
+  }
+}
+```
+
+## CSS Animation
+
+You can use CSS animation to animate both enter and exit animations.
+
```

Comment:
## CSS animations

Use the following Base UI attributes for creating CSS animations when a component becomes visible or hidden:

- `[data-open]` corresponds to the style applied when a component becomes visible
- `[data-ending-style]` corresponds the style applied before a component becomes hidden

Reply:
You'll need to update your CSS example too (it won't work)